### ISSUE TITLE
Next release: focused on A11y

### ISSUE THREAD

**Issue by jeffschram** [crushlovely/skyline#13]
Next release will focus on aria roles and attributes in example markup, as well as begin work on accessible UI components.

Using this issue to store some references:

Heydon Pickering's work:
http://heydonworks.com/practical_aria_examples/
https://vimeo.com/109912253

http://substantial.com/blog/2014/07/22/how-i-audit-a-website-for-accessibility/

**Comment by jeffschram** [comment]
Going to continue this brain dump on ui components to focus on:

### Icons

make sure icons are either aria-hidden if we don't want them spoken, or maybe even better, icons should have descriptive text if appropriate. Use aria-label or aria-labelledby

### Buttons

buttons will be real buttons

### Button Group

swhat is the best way to group buttons? Lists? Toolbar? And what is the best way of handling placement variations? Left, Center, Right, Justified etc.

### Dropdowns

the aria attributes will determine the functionality. Titles and groups of options within the dropdown would be nice. A dropdown is essentially a collapsible where the collapsing content is absolutely positioned. Maybe we can use those similarities to our advantage... simplify.

### Tooltips & Popovers

use aria attributes to define the functionality. These are both very similar components; a popover usually has more content than a tooltip and that's pretty much the major difference.

### Tabs & Carousels

aria attributes define the functionality. What's the difference between a tab set and a carousel? The animation? Prev/Next buttons? Autoplay (I hate autoplay though)

**Comment by aminur** [comment]
Is this project still maintained?

**Comment by jeffschram** [comment]
Hi @aminur - we are primarily focusing on Skyline internally (at Crush & Lovely) and are not actively seeking contributions to the repo. However, ideas and PRs are always welcome.

**Comment by equinusocio** [comment]
This framework is outdated. Is this abandoned?
